api/tests/test_hyp3_api.py

```diff
@@ -5,8 +5,8 @@
 import pytest
 from botocore.stub import Stubber
 from flask_api import status
-from hyp3_api import STEP_FUNCTION_CLIENT, auth, connexion_app
-
+from moto import mock_dynamodb2
+from hyp3_api import DYNAMODB_RESOURCE, auth, connexion_app, STEP_FUNCTION_CLIENT # noqa imports must be in this order for mock db
```

Comment:
technically should be `DYNAMODB_RESOURCE, STEP_FUNCTION_CLIENT, auth, connexion_app`